gym-roboy/envs/ros_proxy.py

```diff
@@ -60,42 +60,67 @@ def __init__(self):
         self.node = rclpy.create_node('gym_client')
         self.step_cli = self.node.create_client(GymStep, 'gym_step')
         self.reset_cli = self.node.create_client(GymReset, 'gym_reset')
+        self.step_size = 1.0;
     
     def forward_reset_command(self):
-        step_size = 1.0
-        req = GymStep.Request()
-        res = GymStep.Response()
-        req.step_size = step_size
+
+        while not self.reset_cli.wait_for_service(timeout_sec=1.0):
+            self.node.get_logger().info('service not available, waiting...')
+
+        req = GymReset.Request()
+
         future = self.reset_cli.call_async(req)
         rclpy.spin_until_future_complete(self.node,future)
         if future.result() is not None:
-            self.node.get_logger().info("result: %f" % future.result().q[1])
-        q = future.result()
-        return MsjRobotState(joint_angle=q.q, joint_vel=q.qdot)
+            q = future.result().q
```

Comment:
this logging logic is duplicated in the other methods. Please put it into one method. 
```python
self._log_robot_state(robot_state: MsjRobotState):
	     qd = future.result().qdot
	     q_str = str(q).strip('[]')
             qd_str = str(qd).strip('[]')
             self.node.get_logger().info("Step, joint angles: %s" % q_str)
             self.node.get_logger().info("Step, joint velocity: %s" % qd_str)
```

Reply:
I will do it now